### ISSUE TITLE
Add figures to all examples in docs

### ISSUE THREAD

**Issue by adishavit** [shapely/shapely#1882]
The Shapely docs are great.  
MIT would be even better if the example snippets had an accompanying figure for every example.  
Is there any reason this is not so?

**Comment by jorisvandenbossche** [comment]
I fully agree that would be nice, but the main reason being that adding illustrations is also quite some work, and nobody started that for now. But contributions are certainly welcome!
